server/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceType.java

```diff
@@ -30,7 +30,8 @@
     ANY,
     NUMERIC,
     BYTES,
-    GEOPOINT;
+    GEOPOINT,
+    RANGE;
```

Comment:
Just a thought: do you know if we have a test anywhere that validates these enum orderings don't change?  We have a test class for that (`AbstractWriteableEnumTestCase`), but I think there are some custom one-off implementations too.

If we don't have a test, might be a nice time to add one just to ensure these don't get broken in the future.

Reply:
I added a note about this in #42949 (aka the ValuesSource refactor ticket).  I don't think this PR is the best place to add a serialization test for this enum though.  If you think it's high urgency, I'm happy to open a different PR against master to address it, or deal with it as part of the refactor.

Reply:
:+1: Let's do it as a separate PR.  I don't think it's super urgent, but untested enums make me nervous so doing it as a separate PR before tackling the refactor is probably a good idea.